### PR TITLE
added partner filtering on tiles page and parter recognition on proje…

### PR DESCRIPTION
**Pull request by malawi-mike**
…ct pages

### CODE REVIEW

File: project.php [hunk 1]
```diff
@@ -143,12 +143,12 @@
 <script type="text/javascript" src="js/imagelightbox2.js"></script>
 <div class="container">
 	
-		<div><h4 class="header left brown-text text-lighten-2 text-shadow: 2px 2px 7px #111111">
-					  <a href='<?php print $mapFilename; ?>' 
-            data-imagelightbox="map<?php print $projectId; ?>" style='font-weight:bold;color:#654321'><?php print $villageName; ?> Village</a>  
+		<div><h4 class="header left brown-text text-lighten-2 text-shadow: 2px 2px 7px #111111" style="padding: 0 0 2% 0">
```
Comment (jdepree) [inline project.php]: Move style attributes (text-shadow) to style tag.

File: project_tiles.php [hunk 2]
```diff
@@ -8,14 +8,25 @@
...
-			</h5>
-		</div>
-		<script>var statusFilter = 0, typeFilter = 0;</script>
-		<div class="col s12 m4 l4 valign-wrapper" style="vertical-align: middle; height:50px; padding:1% 1% 1% 1%;">		
+	<div class="row" style="padding:3% 1% 1% 1%;">
+	<script>var statusFilter = 0, typeFilter = 0;</script>
+		<div class="col s12 m4 l4 center-align valign-wrapper" style="vertical-align: middle; height:50px; padding:1% 1% 1% 1%;">		
+	
+    		    <!-- Dropdown Trigger -->
+      		<a class="dropdown-trigger btn donor-background" style="display: block; margin: 0 auto;" href='#' data-target='dropdown3' id='statusFilter'>Filter by Funder</a>
```
Comment (jdepree) [inline project_tiles.php]: Change Funder to Partner?

File: project_tiles.php [hunk 3]
```diff
@@ -8,14 +8,25 @@
...
+
+            <!-- Dropdown Structure -->
+          	<ul id="dropdown3" class="dropdown-content" style="margin-left:25px;margin-top:50px;">
```
Comment (jdepree) [inline project_tiles.php]: Looks like currently, changing status dropdown causes new value to show up in this one.

File: project_tiles.php (hunk 2 above)
Comment (malawi-mike) [reply project_tiles.php]: probably a good idea but I didn't want to confuse funding parters and village partners.

File: project_tiles.php (hunk 3 above)
Comment (malawi-mike) [reply project_tiles.php]: yeah, don't know what's happening there. any ideas?

Comment (malawi-mike) [reply project_tiles.php]: fixed it

File: project_tiles.php (hunk 2 above)
Comment (malawi-mike) [reply project_tiles.php]: changed it

**Pull request merged**